x/staketia/keeper/unbonding.go

```diff
@@ -56,10 +60,10 @@ func (k Keeper) RedeemStake(ctx sdk.Context, redeemer string, stTokenAmount sdkm
 
 	// Estimate a placeholder native amount with current RedemptionRate
```

Comment:
Doesn't `RedeemStake` need to decrement `RemainingDelegatedBalance`?

Reply:
great eye! yes it should have been decremented - the auditors caught it as well! https://github.com/Stride-Labs/stride/pull/1223/commits/def6f4a65f2b7345774adbfc23f3d6b46e23385f

Reply:
Nice! That fix looks good to me.

Reply:
Good stress test of the auditors (: